docs/concepts/packages.md

```diff
@@ -3,111 +3,103 @@ id: packages
-  * As you can see from the [code](https://github.com/gnolang/gno/blob/master/examples/gno.land/p/demo/grc/grc721/basic_nft.gno#L341), the concept of `Safe` has yet to be implemented.
-* `SetApprovalForAll`: Approves all tokens owned by the `owner` to an `operator`.
-  * You may not set multiple `operator`s.
-* `GetApproved`: Returns the `address` of the `operator` for a token, specified with its `ID`.
-* `IsApprovedForAll`: Returns if all NFTs of the `owner` have been approved to the `operator`.
-
-## `testutils`
+View the package on-chain [here](https://gno.land/p/demo/ufmt), or on GitHub, 
+[here](https://github.com/gnolang/gno/tree/master/examples/gno.land/p/demo/ufmt).
 
-The `testutils` package contains a set of functions that comes in handy when testing realms. The sample function below is the commonly used `TestAddress` function that generates a random address.
+### Package `seqid`
+Deployed under `gno.land/p/demo/seqid`, the `seqid` package provides a simple 
+way to have sequential IDs in Gno. Its encoding scheme is based on the `cford32`
+package. From [`seqid.gno`](https://gno.land/p/demo/seqid/seqid.gno):
 
-[embedmd]:# (../assets/explanation/packages/pkg-3.gno go)
 ```go
-func TestAddress(name string) std.Address {
-	if len(name) > std.RawAddressSize {
-		panic("address name cannot be greater than std.AddressSize bytes")
-	}
-	addr := std.RawAddress{}
-	// TODO: use strings.RepeatString or similar.
-	// NOTE: I miss python's "".Join().
-	blanks := "____________________"
-	copy(addr[:], []byte(blanks))
-	copy(addr[:], []byte(name))
-	return std.Address(std.EncodeBech32("g", addr))
-}
+// Package seqid provides a simple way to have sequential IDs which will be
+// ordered correctly when inserted in an AVL tree.
+//
+// Sample usage:
+//
+//	var id seqid.ID
+//	var users avl.Tree
+//
+//	func NewUser() {
+//		users.Set(id.Next().String(), &User{ ... })
+//	}
+package seqid
 ```
 
-The code takes the `name` as the input and creates a random address. Below is a list of examples where it's used in the test case of the `foo20` realm.
-
-[embedmd]:# (../assets/explanation/packages/pkg-4.gno go)
-```go
-admin := users.AddressOrName("g1tntwtvzrkt2gex69f0pttan0fp05zmeg5yykv8")
-test2 := users.AddressOrName(testutils.TestAddress("test2"))
-recv := users.AddressOrName(testutils.TestAddress("recv"))
-normal := users.AddressOrName(testutils.TestAddress("normal"))
-owner := users.AddressOrName(testutils.TestAddress("owner"))
-spender := users.AddressOrName(testutils.TestAddress("spender"))
-recv2 := users.AddressOrName(testutils.TestAddress("recv2"))
-mibu := users.AddressOrName(testutils.TestAddress("mint_burn"))
-```
+View the package on-chain [here](https://gno.land/p/demo/seqid), or on GitHub,
+[here](https://github.com/gnolang/gno/tree/master/examples/gno.land/p/demo/seqid).
+
+## Packages vs Standard Libraries
+Apart from packages, Gno, like Go, has standard libraries. To better
```

Comment:
There is only *one* standard library, containing multiple packages, either in Go or Gno.